assets/i18n/zh-tw.json

```
@@ -0,0 +1,269 @@
+    "dgc-windows-wallet-modal-ledger": "這代表，您必須透過 \"ledger\" 使用狗狗幣", 
+    "dgc-windows-wallet-modal-light-wallet": "MultiDoge 是輕量化的錢包，利用「Skimming」的方式更快地同步區塊鏈。",
+    "dgc-windows-wallet-modal-full-wallet": "Dogecoin Core 是「完整」的錢包，利用完整下載的方式同步區塊鏈，成為功能強大的狗狗幣錢包。", 
+    "dgc-windows-wallet-modal-prosandcons": "優點與缺點？", 
+    "dgc-windows-wallet-modal-different-strokes": "兩種錢包都有優點與缺點", 
+    "dgc-windows-wallet-modal-multidoge-pro": "MultiDoge 是輕量化的錢包，只需要區塊鏈上「瀏覽」即可使用，這非常快！", 
+    "dgc-windows-wallet-modal-multidoge-con": "然而這樣的錢包僅能在一般用途使用，<strong>無法</strong>進行挖礦來獲取狗狗幣，否則將使錢包變得緩慢", 
+    "dgc-windows-wallet-modal-dogecoin-core-con": "Dogecoin Core 將會下載整個狗狗幣區塊鏈，與 MultiDoge 相比，第一次使用將會非常地緩慢且佔許多硬碟空間。", 
+    "dgc-windows-wallet-modal-dogecoin-core-pro": "然而它有優點，可以利用此錢包安全地進行狗狗幣挖礦，因為有能力可以處理交易，也有助於維護狗狗幣的區塊鏈網路", 
+    "dgc-osx-wallet-modal-dogecoin-core": "Dogecoin Core.", 
+	"dgc-android-wallet-modal-title": "從 GOOGLE PLAY 下載.",
+    "dgc-android-wallet-modal-apk": "<a href=\"https://github.com/langerhans/dogecoin-wallet-new/releases/download/1.08/wallet-1.08-aligned.apk\">Direct APK Download.</a>",
+	"dgc-ios-wallet-modal-title": "從 APP STORE 下載.",
+    "dgc-getting-started-title": "開始吧",
+    "dgc-getting-started-desc": "開始使用狗狗幣",
+    "dgc-getting-started-is-123": "這很 <strong>簡單</strong> 只要 1, 2, 3 步驟即可!",
+    "dgc-getting-started-desktop": "電腦桌面錢包",
+    "dgc-getting-started-phone": "手機錢包",
+    "dgc-getting-started-online": "線上錢包",
+    "dgc-getting-started-paper": "紙錢包",
+    "dgc-getting-started-choose": "選擇一種錢包開始",
+    "dgc-getting-started-desktop-os": "Please choose your Computer's Operating System.",
+    "dgc-getting-started-phone-os": "Please choose your Smartphone's Operating System.",
+    "dgc-getting-started-online-choice": "Please choose your Online Wallet.",
+    "dgc-getting-started-walletgen": "WalletGenerator",
+    "dgc-getting-started-paper-choose": "Please choose your Paper Wallet.",
+    "dgc-getting-started-1-0": "１、下載錢包",
+    "dgc-getting-started-windows-1-1": "點選下方連結下載狗狗幣錢包 MultiDoge <br/>如果被詢問是否允許，請允許下載",
+    "dgc-getting-started-windows-1-2": "當完成檔案下載開啟 (.exe) 檔案",
+    "dgc-getting-started-windows-1-3": "下載 Windows 版本 MultiDoge",
+    "dgc-getting-started-2-0": "２、安裝錢包",
+    "dgc-getting-started-windows-2-1": "在安裝過程中，當您被詢問許可協議請接受並繼續安裝。",
+    "dgc-getting-started-windows-2-2": "完成安裝後，找到 MultiDoge 捷徑並開啟",
+    "dgc-getting-started-3-0": "３、使用錢包！",
+    "dgc-getting-started-windows-3-1": "啟動後，您的錢包將會在左下方出現等待進度條，您必須等待 1 ~ 5 分鐘讓錢包同步。",
+    "dgc-getting-started-windows-3-2": "錢包與區塊鏈網路同步後，您就可以開始使用狗狗幣錢包了！",
+    "dgc-getting-started-wallet-done": "您已經完成狗狗幣錢包的設置！",
+    "dgc-getting-started-congrats": "<strong>恭喜！</strong> 讓我們好好地使用吧！",
+    "dgc-getting-started-windows-acknowledge": "Windows 指引與圖片創建者 <a href=\"https://reddit.com/user/PowerLemons\" target=\"_blank\">/u/PowerLemons</a>.",
+    "dgc-getting-started-osx-1-1": "點選下方連結下載狗狗幣錢包 MultiDoge",
+    "dgc-getting-started-osx-1-2": "完成下載後，在下拉選單中的\"下載\"項目，開啟檔案。",
```

Original "Once the download is finished, double click on the file in the \"Downloads\" drop down menu."
```suggestion
    "dgc-getting-started-osx-1-2": "當完成下載後，雙擊在下拉選單中的「下載」中的檔案開啟程式。",
```